doc/source/whatsnew/v0.21.0.txt

```diff
@@ -179,6 +179,7 @@ Sparse
 ^^^^^^
 
 
+- Bug in instantiating :class:`SparseSeries` from ``dict`` with or without ``index=`` kwarg (:issue:`16905`)
```

Comment:
It's a reference to the _two issues_ fixed. With, the result was invalid; without, it crashed.